### DISCUSSION TITLE
JS Interpreter?

### DISCUSSION THREAD

**Discussion by RichardKanshen** [TaserTheFox/EsoFur-Interpreter#28] [Ideas]
Just found this, and I find it very cool. Just wanted to ask if there is a JS Interpreter planned, and if not, if it would be okay to make one.

**Comment by TaserTheFox** [comment]
Heya thank you for showing interest in my little project haha, planned? no, i dont know enough JS to do it, and most of this was written by chatGPT, and embellished by me and lots of googling. as for making one go right ahead just make sure that if you post it anywhere, you link it back to this github page. though on that note, most features are a work in progress, or outright broken/conflicting with other features. feel free to even contribute to the original python one! also the esolang wiki page is probably outdated.

**Reply by RichardKanshen** [reply]
Alright, I will keep this in mind, and thanks for the reply ^^
